### DISCUSSION TITLE
How to Remove a member from a group

### DISCUSSION THREAD

**Discussion by anju979git** [microsoftgraph/msgraph-sdk-go#116] [Q&A] [answered]
Hi,

I didn't find the method to **Remove a member from a group** [(Remove Member)](https://docs.microsoft.com/en-us/graph/api/group-delete-members?view=graph-rest-1.0&tabs=http) in the msgraph sdk using **GoLang**. 

I tried the following, but Delete method is not available  in the latest version(v0.15.0) also :
`err := graphClient.GroupsById(group-id).MembersById(member-id).Delete(nil)`

Could you please point me to an example/documentation where I can know more about this?

Thanks in advance.

**Comment by baywet** [comment, marked as answer]
Hi @anju979git 
Thanks for trying the Go SDK and for reaching out.
This SDK is auto-generated based on metadata, and I believe the metadata is incorrect, hence why this method is missing.
I've [logged an issue in the metadata repo](https://github.com/microsoftgraph/msgraph-metadata/issues/131).
While we fix that, you can use another request builder as a workaround like so.
https://github.com/microsoftgraph/msgraph-sdk-go/blob/7d7ebff89500456fd466788c39810d2f41bb54b3/admin/serviceannouncement/service_announcement_request_builder.go#L76

```golang
requestBuilder := NewServiceAnnouncementRequestBuilder("https://graph.microsoft.com/v1.0/groups/{group-id}/members/{members-id}/$ref", requestAdapter)
requestBuilder.Delete(nil)
```

**Reply by anju979git** [reply]
Above snippet served the purpose. Thanks.

msgraph sdk is quite useful. When can we expect the final release (any probable dates)?

**Reply by baywet** [reply]
ideally by June, but that's a tentative, not a promise, since we don't have anyone staffed on this.
